jobs/cjd_sample/README.md

```diff
@@ -0,0 +1,54 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google
```

Comment:
I recommend borrowing a bit from this [README.md](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/vision/README.md).